### ISSUE TITLE
Device State/backup restore

### ISSUE THREAD

**Issue by dougK44883** [jessedp/tablo-tools-electron#25]
There doesn't appear to be a facility to respond or comment to a card in the backlog  Feature Ideas
#### Expected Behavior
You may already be aware of this, there's a project - https://github.com/rchavers/backup-tablo https://community.tablotv.com/t/backup-tablo-sh-backup-all-recorded-shows-from-a-tablo/21267
which "inadvertently" backups tablos DB - used when migrating to a new device.  He uses lftp, have have not been able to navigate to it, but the script does back it up.  If this may be something similar to what you have in mind, it might be helpful.

#### Current Behavior
Add "don't overwrite" option for Exports

Also suggest renaming option.  In the event of a segmented recording, you want/need each piece.  Past experience,  overlooked a previous "bad" recording, so if I exported another copy renamed - one might be full length other not... or both could be good and I only need one.
#### Steps to Reproduce (for bugs)

1.

2.

3.

4.

#### Your Environment

- App version -
- Last 4 of Tablo ServerID -
- Tablo Firmware version -
- Operating System and version -

**Comment by jessedp** [comment]
TL;DR - Please open 1 Issue for each thing that needs to be tracked/discussed/you want updates on and I'll deal with any "duplicates".

Right - the Backlog and whatnot are just in a project management tool for maintainers (me) to work in. The idea is that for each problem/feature an Issue (like this) is created. Then I can pull it into wherever it needs to be in the Backlog or a Release and talk about whatever here.

Since a lot of the stuff there I pulled out of the forum and had in a text file, there are no Issues for them yet - _I'm_ planning on only converting them to Issues once I start working on something. But if you or someone else opens the issue I've listed, I'll just replace my entry with the Issue so the same thing is captured and we now have somewhere to track/discuss

I'm going to close this and turn it into 2 issues. And look at my issue templates...
